packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -726,7 +726,11 @@ module.exports = {
       type: 'string',
       enum: ['development', 'production'],
     },
+    homeDirHash: {
```

Comment:
It must be in ConfigFile, not in the specific config. It's gonna be the same for all configs. Let's call it `projectId`. And it must always set (can't be null).

Reply:
I realized that to ensure backward compatibility with the current version (people already started nodes with 24). We should make it optional and use the old compose project name if it's not set. When we create a new config.json we should add it if we use the old one then it will use legacy naming.